package.json

```diff
@@ -1,85 +1,82 @@
 {
-  "name": "@r3f-perf/root",
-  "version": "0.0.0-semantic-released",
+  "name": "r3f-perf",
+  "version": "7.0.0",
+  "description": "Easily monitor your ThreeJS performances.",
+  "keywords": [
+    "react",
+    "three",
+    "benchmark",
+    "performance",
+    "monitor",
+    "analysis",
+    "profiling",
+    "r3f"
+  ],
+  "author": "Renaud ROHLINGER (https://github.com/RenaudRohlinger)",
+  "homepage": "https://github.com/utsuboco/r3f-perf",
+  "repository": "https://github.com/utsuboco/r3f-perf",
   "license": "MIT",
-  "private": true,
-  "workspaces": [
-    "packages/*",
-    "demo"
+  "files": [
+    "dist/*",
+    "src/*"
   ],
-  "preconstruct": {
-    "packages": [
-      "packages/*"
-    ]
+  "types": "./dist/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs"
+    },
+    "./headless": {
+      "types": "./dist/headless.d.ts",
+      "require": "./dist/headless.js",
+      "import": "./dist/headless.mjs"
+    }
   },
```

Comment:
For modern tooling and Node versions, any paths in `exports` are the only valid paths to import from. In ESM-based tooling, this would otherwise require a file extension for imports which is very error-prone (e.g. https://github.com/mrdoob/three.js/issues/24593). This also plagues React and its JSX transform between 17-18 since the introduction of `package.exports` was a breaking change, in this regard.